src/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Tools.Common;
+using Microsoft.DotNet.Tools.NuGet;
+
+namespace Microsoft.DotNet.Tools.List.PackageReferences
+{
+    internal class ListPackageReferencesCommand : CommandBase
+    {
+        //The file or directory passed down by the command
+        private readonly string _fileOrDirectory;
+        private AppliedOption _appliedCommand;
+
+        public ListPackageReferencesCommand(
+            AppliedOption appliedCommand,
+            ParseResult parseResult) : base(parseResult)
+        {
+            if (appliedCommand == null)
+            {
+                throw new ArgumentNullException(nameof(appliedCommand));
+            }
+
+            // Gets the absolute path of the given path
+            _fileOrDirectory = PathUtility.GetAbsolutePath(PathUtility.EnsureTrailingSlash(Directory.GetCurrentDirectory()),
+                                                           appliedCommand.Arguments.Single());
+
+            FileAttributes attr = File.GetAttributes(_fileOrDirectory);
```

Comment:
Is there a reason `File.GetAttributes` is used here instead of `Directory.Exists`?  The former throws exceptions for conditions such as invalid path or file not found.  If I do `dotnet list packages foo.csproj`, what is happening with this feature currently if `foo.csproj` does not exist?

Reply:
Also, is there a reason we need to format this here with a trailing backslash if the path is a directory?